src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -132,8 +132,29 @@ trait IterableOps[+A] extends Any {
   }
 
   override def toString = s"$className(${mkString(", ")})"
+
+  def sameElements[B >: A](that: Iterable[B]): Boolean = {
```

Comment:
You’re right, that makes no sens to inherit from that in `Set`. Is it ok to remove it from here? (it will not be backward compatible)